javav2/example_code/pinpoint/Readme.md

```diff
@@ -91,35 +130,36 @@ For the purpose of the JUnit tests, the **ImportSegments** test uses a JSON file
    	 }
 	 }
 
+
 ### Command line
 To run the JUnit tests from the command line, you can use the following command.
 
-	mvn test
+		mvn test
 
 You will see output from the JUnit tests, as shown here.
 
-	 [INFO] -------------------------------------------------------
-	 [INFO]  T E S T S
-	 [INFO] -------------------------------------------------------
-	 [INFO] Running AmazonPinpointTest
-	 Running Amazon Test 1
-	 Running Amazon Test 2
-	 ...
-	 Done!
-	 [INFO] Results:
-	 [INFO]
-	 [INFO] Tests run: 17, Failures: 0, Errors: 0, Skipped: 0
-	 [INFO]
-	 INFO] --------------------------------------------
-	 [INFO] BUILD SUCCESS
-	 [INFO]--------------------------------------------
-	 [INFO] Total time:  12.003 s
-	 [INFO] Finished at: 2020-02-10T14:25:08-05:00
-	 [INFO] --------------------------------------------
+	[INFO] -------------------------------------------------------
+	[INFO]  T E S T S
+	[INFO] -------------------------------------------------------
+	[INFO] Running DynamoDBTest
+	Running Amazon DynamoDB   Test 1
+	Running Amazon DynamoDB  Test 2
```

Comment:
Same question, is the DynamoDBTest example intended to be here in the Pinpoint README?